src/full.rs

```diff
@@ -154,120 +154,51 @@ pub fn transmute_many_permissive<T: TriviallyTransmutable>(bytes: &[u8]) -> Resu
 ///            &[0x0E0F, 0x0B0A]);
 /// # }
 /// ```
-pub fn transmute_many_pedantic<T: TriviallyTransmutable>(bytes: &[u8]) -> Result<&[T], Error> {
+pub fn transmute_many_pedantic<T: TriviallyTransmutable>(bytes: &[u8]) -> Result<&[T], Error<u8, T>> {
     transmute_many::<T, PedanticGuard>(bytes)
 }
 
 /// Transform a byte vector into a vector of values.
```

Comment:
A leftover? ("bytes")

Reply:
We _are_ transmuting bytes here, right?

Reply:
Not necessarily, as the argument is `Vec<S: TriviallyTransmutable>`? Bit confusing tbh

Reply:
Derp, I wasn't looking at the right line.